optuna/terminator/improvement/gp/botorch.py

```diff
@@ -55,10 +55,11 @@ def fit(
         y = torch.tensor([trial.value for trial in trials], dtype=torch.float64)
         y = torch.unsqueeze(y, 1)
 
+        noise = 1e-8 * y.std().item()
         self._gp = FixedNoiseGP(
             x,
             y,
-            torch.full_like(y, 1e-8),
+            torch.full_like(y, noise),
```

Comment:
Almost LGTM. But how about writing in one line?
```suggestion
            torch.full_like(y, 1e-8 * y.std().item()),
```

Reply:
This actually takes [the follow-up PR](https://github.com/optuna/optuna/pull/4490/files) into consideration, where there would be too many things in one line if we [get the min noise value from the gpytorch setting](https://github.com/optuna/optuna/pull/4490/files#diff-82cb56a8e78506746061bbf40a9f6b1852cdd6a93406348a3bb1b3b4c6f8a05eR63). 

Anyway, I think writing one line is much simpler so far in this PR. Thanks for your suggestion!